src/OpenApi/SchemaDefinition/Entity/EntityAdapter.php

```diff
@@ -261,18 +261,19 @@ protected function normalizeType(string $type): string
 
 	private function getNativePropertyType(Type $type, ReflectionProperty $property): string
 	{
-		if ($type->isSingle() && count($type->getNames()) === 1) {
-			return $type->getNames()[0];
+		if ($type->isSingle() && count($type->getNames()) === 1) { /** @phpstan-ignore-line */
```

Comment:
Proc je tu vsude phpstan ignore? Predtim tu nebyl.

Reply:
David to dal jako deprecated, zkousel jsem udelat isSimple() jak je doporucovane, ale testy pro php8.0 neprosly.

Reply:
Asi to nebude nejlepsi reseni, protoze kdyz se ta metoda smaze, tak to stejne phpstan nenahlasi.. zkusim nejak udelat podminky kde pridam pro overeni i verzi php? To jsem zkousel taky ale neprochazelo asi ze jsem tam mel or mezi tim, tak bych to musel udelat jako 2 separatni code blocky, jeden pro php8.0 a dalsi pro php8.1+.

Reply:
Nemuzu pouzit novou metodu ani s php8 podminkou https://github.com/contributte/apitte/actions/runs/4217891774/jobs/7321994500, takze jsem vratil starou a presunul deprecated warning do phpstan.neon